packages/admin-ui/src/lib/core/src/shared/components/asset-preview/asset-preview.component.ts

```diff
@@ -33,6 +33,7 @@ export type PreviewPreset = 'tiny' | 'thumb' | 'small' | 'medium' | 'large' | ''
 })
 export class AssetPreviewComponent implements OnInit, OnDestroy {
     @Input() asset: AssetLike;
+    @Input() assets: AssetLike[];
```

Comment:
mark this as optional since existing external uses of the component will not be passing it.

```ts
@Input() assets?: AssetLike[];
```